pkg/logs/processor/processor.go

```diff
@@ -18,16 +18,18 @@ import (
 type Processor struct {
 	inputChan       chan *message.Message
 	outputChan      chan *message.Message
+	diagnosticChan  chan message.Message
```

Comment:
Yeah - the message `Content` gets re-written when it gets encoded so its safer to keep a copy. 
1. so we don't risk any concurrency issues
2. since the un-encoded `Content` is just the log line in plaintext.